x-pack/plugins/security_solution/public/timelines/store/timeline/epic.ts

```diff
@@ -105,13 +103,11 @@ const timelineActionsType = [
   updateDataProviderExcluded.type,
   updateDataProviderKqlQuery.type,
   updateDataProviderType.type,
-  updateDescription.type,
```

Comment:
why we don't want to listen on `updateTitle` and `updateDescription`?

Reply:
If I understand it correctly, the list is for auto-saved actions.